index.js

```diff
@@ -1 +1,281 @@
 'use strict';
+
+var handlebars = require('handlebars'),
+	read = require('fs').readFileSync,
+  write = require('fs').writeFile;
+
+// var swag = require('./test/minimal/swagger.json');
+
+// var config = {
+//     'assertionFormat': 'should',
+//     'pathNames': ['/', '/user'],
+//     'testmodule': 'supertest',
+//     'destination': './test/minimal'
+//   };
+
+/**
+ * Builds a unit test stubs for the response code of a path's operation
+ * @param  {json} swagger swagger file containing API
+ * @param  {string} path API path to generate tests for
+ * @param  {string} operation operation of the path to generate tests for
+ * @param  {string} response response type of operation of current path
+ * @param  {json} config configuration for testGen
+ * @returns {string} generated test for response type
+ */
+function testGenResponse(swagger, path, operation, response, config) {
+  var result, gen, source, param,
+    // request payload
+    data = {
+      'responseCode': response,
+      'description': swagger.paths[path][operation].responses[response].description,
+      'assertion': config.assertionFormat,
+      'asynchronous': true
+    };
+
+  // adding body parameters to payload
+  if (swagger.paths[path][operation].hasOwnProperty('parameters')) {
+    data.parameters = [];
+
+    // only adds body parameters to request, ignores query params
+    for (param in swagger.paths[path][operation].parameters) {
+      if (swagger.paths[path][operation].parameters[param].in === 'body') {
+        data.parameters.push(swagger.paths[path][operation].parameters[param]);
+      }
+    }
+  }
+
+  // request url vs. supertest path
+  if (config.testmodule === 'request') {
+    data.url = swagger.schemes[0] + '://' + swagger.host +
```

Comment:
There is no guarantee that `swagger.schemes[0]` exists

Same for `host`. Default to `localhost`